### PR TITLE
add get/setOperatorParametersExtended3 commands

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/mtx/commands/downlink/getOperatorParametersExtended3.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ *
+ * const bytes = getOperatorParametersExtended3.toBytes();
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // output:
+ * [113, 0]
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/getOperatorParametersExtended3.md#request)
```
Comment (DarkPark) [inline src/mtx/commands/downlink/getOperatorParametersExtended3.ts:18]:
```suggestion
 * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetOperatorParametersExtended3.md#request)
```

File: src/mtx/commands/uplink/getOperatorParametersExtended3.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ *     relaySet: {
+ *         RELAY_OFF_LIMIT_P_MINUS_T1: true,
+ *         RELAY_OFF_LIMIT_P_MINUS_T2: false,
+ *         RELAY_OFF_LIMIT_P_MINUS_T3: true,
+ *         RELAY_OFF_LIMIT_P_MINUS_T4: false
+ *     }
+ * }
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/getOperatorParametersExtended3.md#response)
```
Comment (DarkPark) [inline src/mtx/commands/uplink/getOperatorParametersExtended3.ts:34]:
```suggestion
 * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/GetOperatorParametersExtended3.md#response)
```

File: src/mtx/commands/uplink/setOperatorParametersExtended3.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+ *
+ * // decoded payload
+ * const parameters = setOperatorParametersExtended3.fromBytes(bytes);
+ *
+ * console.log(parameters);
+ * // output:
+ * {}
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/setOperatorParametersExtended3.md#response)
```
Comment (DarkPark) [inline src/mtx/commands/uplink/setOperatorParametersExtended3.ts:23]:
```suggestion
 * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx/commands/SetOperatorParametersExtended3.md#response)
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -56,7 +56,7 @@
         "build": "npm run build:js && npm run build:docs",
         "build:clean": "npm run build:js:clean && npm run build:docs:clean",
         "lint:tsc": "tsc --project ./tsconfig.eslint.json --noEmit",
-        "lint:eslint": "eslint ./src ./tests --ext .ts",
+        "lint:eslint": "eslint ./src ./tests --ext .ts --ignore-pattern 'src/test.ts'",
```
[the remark this answers is not in the document]
Comment (DarkPark) [reply package.json:59]: but be careful - after disabling eslint you may accidentally copy-paste bad code to command example

File: src/mtx/commands/downlink/getOperatorParametersExtended3.ts (hunk 1 above)
Comment (aleks-and) [reply src/mtx/commands/downlink/getOperatorParametersExtended3.ts]: fixed in all places

File: src/mtx/commands/uplink/getOperatorParametersExtended3.ts (hunk 2 above)
Comment (aleks-and) [reply src/mtx/commands/uplink/getOperatorParametersExtended3.ts]: fixed in all places

File: src/mtx/commands/downlink/setOperatorParametersExtended3.ts [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+ *         RELAY_OFF_LIMIT_P_MINUS_T4: false
+ *     }
+ * };
+ * const bytes = setOperatorParametersExtended3.toBytes(parameters);
+ *
+ * // command binary representation
+ * console.log(bytes);
+ * // output:
+ * [114, 17, 0, 0, 0, 100, 0, 0, 0, 200, 0, 0, 1, 44, 0, 0, 1, 144, 20]
+ * ```
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/commands/downlink/setOperatorParametersExtended3.ts:28]: fixed in all places

File: src/mtx/commands/uplink/setOperatorParametersExtended3.ts (hunk 3 above)
Comment (aleks-and) [reply src/mtx/commands/uplink/setOperatorParametersExtended3.ts]: fixed in all places

File: src/mtx/commands/uplink/getOperatorParametersExtended3.ts [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+                RELAY_OFF_LIMIT_P_MINUS_T2: false,
+                RELAY_OFF_LIMIT_P_MINUS_T3: true,
+                RELAY_OFF_LIMIT_P_MINUS_T4: false
+            }
+        },
+        bytes: [
+            0x71, 0x11,
+            0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0xc8, 0x00, 0x00, 0x01, 0x2c, 0x00, 0x00, 0x01, 0x90, 0x14
+        ]
+
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/commands/uplink/getOperatorParametersExtended3.ts]: fixed

File: src/mtx/commands/downlink/setOperatorParametersExtended3.ts [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+                RELAY_OFF_LIMIT_P_MINUS_T2: false,
+                RELAY_OFF_LIMIT_P_MINUS_T3: true,
+                RELAY_OFF_LIMIT_P_MINUS_T4: false
+            }
+        },
+        bytes: [
+            0x72, 0x11,
+            0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0xc8, 0x00, 0x00, 0x01, 0x2c, 0x00, 0x00, 0x01, 0x90, 0x14
+        ]
+
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/commands/downlink/setOperatorParametersExtended3.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**